src/main/java/me/dinowernli/grpc/prometheus/Configuration.java

```diff
@@ -66,12 +82,24 @@ public CollectorRegistry getCollectorRegistry() {
     return latencyBuckets;
   }
 
+  /** Returns the configured list of headers to be used as labels. */
+  public List<String> getLabelHeaders() {
+    return labelHeaders;
+  }
+
+  /** Returns the sanitized version of the label headers, after turning all hyphens to underscores. */
+  public List<String> getSanitizedLabelHeaders() {
```

Comment:
Why is this sanitization concept necessary? If someone has added labels with hyphens, can we not leave them as they are?

Reply:
Because prometheus disallows hyphens in label names.